p4include/v1model.p4

```diff
@@ -415,7 +415,7 @@ extern Checksum16 {
  *                   may be supported).  Must be a compile-time
  *                   constant.
  */
-extern void verify_checksum<T, O>(in bool condition, in T data, inout O checksum, HashAlgorithm algo);
+extern void verify_checksum<T, O>(in bool condition, in T data, in O checksum, HashAlgorithm algo);
```

Comment:
If we make this change, we should also make the corresponding change to the signature of function verify_checksum_with_payload.